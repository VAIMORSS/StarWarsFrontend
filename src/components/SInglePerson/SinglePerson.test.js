import React from 'react'
import {shallow} from 'enzyme';
import SinglePerson from './index';
import testPerson from './../../assets/testPerson.json'

describe('Single Person',()=>{
    it('should render correctly in "debug" mode',()=>{
        const component =shallow(<SinglePerson Person={testPerson} debug/>);
        expect(component).toMatchSnapshot();
    })
});
