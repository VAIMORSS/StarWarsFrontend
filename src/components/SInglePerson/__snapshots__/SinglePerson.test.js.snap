// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Single Person should render correctly in "debug" mode 1`] = `
<div
  className="card"
>
  <img
    alt="Card image cap"
    className="card-img-top avatar"
    src="https://starwarsblog.starwars.com/wp-content/uploads/2017/01/sw-the-last-jedi-tall-1200x630.jpg"
  />
  <div
    className="card-body"
  >
    <h5
      className="card-title"
    >
      Luke Skywalker
    </h5>
  </div>
  <div
    className="card-footer"
  >
    <button
      className="btn btn-dark"
      onClick={[Function]}
    >
      More Info
    </button>
  </div>
</div>
`;
